Source/FakeItEasy.Tests/FakeOptionsBuilderTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace FakeItEasy.Tests
+{
+    using System;
+    using FakeItEasy.Creation;
+    using FluentAssertions;
+    using NUnit.Framework;
+    using TestHelpers;
+
+    public class FakeOptionsBuilderTests

```

Comment:
Missing `[TestFixture]` attribute. ReSharper doesn't seem to care and detects the tests anyway, but I'm not sure about the NUnit console runner.